src/include/duckdb/function/aggregate_function.hpp

```diff
@@ -128,7 +128,22 @@ class AggregateFunction : public SimpleFunction {
 		case SQLTypeId::DECIMAL:
 			return UnaryAggregate<double, double, double, OP>(type, type);
 		default:
-			throw NotImplementedException("Unimplemented numeric type");
+			throw NotImplementedException("Unimplemented numeric type for unary aggregate");
+		}
+	}
+
+	template <class OP> static AggregateFunction GetBitfieldUnaryAggregate(SQLType type) {
```

Comment:
Perhaps this could be a static method in `bitagg.cpp`, as it is used only in that file.

Reply:
Can do.